lecture_1/08_jobs/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/sethgrid/pester"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+type application struct {
+	Name string
+	Age int
+	Passed bool
+	Skills []string
+}
+
+const url = "https://run.mocky.io/v3/f7ceece5-47ee-4955-b974-438982267dc8"
+
+func contains(arr []string, item string) bool{
+
+	for _, val := range arr{
+		if val == item{
+			return true
+		}
+	}
+	return false
+
+}
```

Comment:
This is correct, but check [this](https://www.geeksforgeeks.org/string-contains-function-in-golang-with-examples) out.